api/files/serializers.py

```diff
@@ -391,8 +399,8 @@ class FileVersionSerializer(JSONAPISerializer):
     date_created = VersionedDateTimeField(source='created', read_only=True, help_text='The date that this version was created')
     links = LinksField({
         'self': 'self_url',
-        'html': 'absolute_url',
         'download': 'get_download_link',
+        'html': 'get_render_link',
```

Comment:
Replacing old `html` link with this new one, which is a render link

Reply:
Do we need to version this change, since technically it's returning different things in the response?

Reply:
good question! What do you think @brianjgeiger? The old html link went to the file detail page with the file rendered in it, and the new one renders the file by itself.

Reply:
Haha. Okay, remembering why we *actually* need this (so we can have the storage-i18n-imbued link to pass to MFR for rendering), let's instead call these new links 'render', and while we're here we can have an `html` link in `files` that's equivalent to the one in `versions`. Sorry for the confusion.

Reply:
sounds good 👍

Reply:
I broke my own rule about not changing commit history during CR, but all the 'html'-'render' changes were neatly contained in one commit!